src/utils/userop.ts

```diff
@@ -202,19 +175,21 @@ export function deepHexlify(obj: any): any {
 
 export function getAddressFromInitCodeOrPaymasterAndData(
     data: Hex
-): Address | undefined {
+): Address | null {
     if (!data) {
-        return undefined
+        return null
     }
     if (data.length >= 42) {
         return getAddress(data.slice(0, 42))
     }
-    return undefined
+    return null
 }
 
 export const transactionIncluded = async (
+    isVersion06: boolean,
```

Comment:
it's a bit having this a boolean no? surely you'd just want two separate functions, one for each? idk but this seems a bit sus

Reply:
most of the code is re-usable it's just the change of abi so though better to have a boolean.